### PR TITLE
community/ambassadors: Adding Michiel Manten

### PR DESCRIPTION
**Pull request by Thingee**

### CODE REVIEW

**Comment by Thingee** [comment]
https://ambassador-michiel-manten.ceph.io/en/community/ambassadors/

File: src/en/community/ambassadors/index.html [hunk 1]
```diff
@@ -170,6 +170,34 @@ <h3 class="h3" id="joachim-kraftmayer">Joachim Kraftmayer</h3>
...
+            loading="lazy"
+            src="/assets/bitmaps/ambassador-michiel-manten.jpg"
+          />
+          <div>
+            <h3 class="h3" id="Michiel Manten">Michiel Manten</h3>
+            <span class="block p">Region: Benelux (Belgium, Netherlands and Luxemburg)</span>
+            <p class="mb-0 p">
+              With over 25 years of IT experience, Michiel Manten has been
+              working with Fairbanks and 42on since 2012, supporting open
+              source teams in 'high-tech industries' from all over the world.
```
Comment (anthonyeleven) [inline src/en/community/ambassadors/index.html:187]: Take out single quotes

File: src/en/community/ambassadors/index.html [hunk 2]
```diff
@@ -170,6 +170,34 @@ <h3 class="h3" id="joachim-kraftmayer">Joachim Kraftmayer</h3>
...
+              Together with Linux, OpenStack and Kubernetes, Ceph is the major
+              focus point of the organisations. Michiel has seen all kinds of
```
Comment (anthonyeleven) [inline src/en/community/ambassadors/index.html:189]: Organizations

File: src/en/community/ambassadors/index.html [hunk 3]
```diff
@@ -170,6 +170,34 @@ <h3 class="h3" id="joachim-kraftmayer">Joachim Kraftmayer</h3>
...
+              Ceph clusters, ranging from terabytes to petabytes, spinning
+              disks or ‘all flash’, CephFS, RBD and S3. He was involved with
```
Comment (anthonyeleven) [inline src/en/community/ambassadors/index.html:191]: Quotes—

File: src/en/community/ambassadors/index.html [hunk 4]
```diff
@@ -170,6 +170,34 @@ <h3 class="h3" id="joachim-kraftmayer">Joachim Kraftmayer</h3>
...
+              organising parts of the OpenInfra Summit, OpenStack Days, Ceph
+              Days, CloudStack Ceph Days and Cephalocon events and he is always
+              looking forward to keeping Ceph alive and active both in the
+              Benelux region as worldwide.
```
Comment (anthonyeleven) [inline src/en/community/ambassadors/index.html:195]: s/as/and/

**Review by anthonyeleven — APPROVED** [review]
Several suggestions, approved in advance to streamline.

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
In docs we tyrannically assert US English, so why not here? ;)

**Pull request merged**